js/google_ima.js

```diff
@@ -272,6 +274,11 @@ require("../html5-common/js/utils/utils.js");
       this.registerUi = function()
       {
         this.uiRegistered = true;
+        if (_amc.ui.useSingleVideoElement && !this.sharedVideoElement && _amc.ui.ooyalaVideoElement[0] &&
+            (_amc.ui.ooyalaVideoElement[0].className === "video")) {
```

Comment:
Oh this is actually a property that native to DOM elements.  We could also do a get on the property but it would be something like getProperty("className") and I don't know if that's better than reading it directly.  Can you explain what you mean by fetched?